### ISSUE TITLE
PyQt5 is only compatible with Python 3

### ISSUE THREAD

**Issue by lucas993** [Sterncat/zernikeapp#1]
It appears?  Are you using PyQt4 or PyQt5?

**Comment by Sterncat** [comment]
PyQt4, and I just learn how to put matplotlib plot in PyQt4.

**Comment by ghost** [comment]
Have you looked at [PyQtGraph](http://www.pyqtgraph.org/) ?  I had really easy success with it recently, and it's much simpler to customize and control than matplotlib.  There are lots of demo scripts.

**Comment by Sterncat** [comment]
I checked the PyQtGraph before, it still needs PyQt4.8+. So for
convenience I chose the PyQt4. For now, PyQt4 could easily
embedded matplotlib in it.

2016-09-13 9:16 GMT-07:00 Adam Hayes notifications@github.com:

**Comment by lucas993** [comment]
Looks like panel.py is using PyQt5.  Maybe this works with Python 3 rather than Python 2.7?

`from PyQt5.QtWidgets import (QWidget, QGridLayout, QLineEdit, QLabel, QHBoxLayout,
    QVBoxLayout, QPushButton, QApplication)`
`from PyQt5.QtGui import QPixmap`
`from PyQt5 import QtCore`

> Zernike Polynomials Application(PyQt Implement)
> 
> How to use
> 
> Yo need following python module:
> 
> numpy, matplotlib, PyQt5
> 
> cd in the zernikeappp directory

**Comment by Sterncat** [comment]
Oh, my wrong. It use pyqt5.But the core of zernikeapp has sime small
problem which doesn't work well under python3. If you want to, you could
easily fix it.

**Comment by lucas993** [comment]
Yeah, I could take a shot.  Do you have the problem isolated?  Or any notes?

**Comment by Sterncat** [comment]
If I have time, I would rewrite it and make it a real exe file. That
version will use PyQt4,python2.7 and user don't need python in their
system. User could just download and run it. So if you really want it now,
you could give a shot. Otherwise, I will just abandond this version.
